src/api/dutchx.ts

```diff
@@ -10,9 +10,9 @@ let dutchXInst: any
 export const initDutchXConnection = async (DUTCHX_OPTIONS: any) => {
   console.log(' ===> FIRING initDutchX ACTION')
   try {
-    dutchXInst = await dutchX.init(DUTCHX_OPTIONS)
-    return dutchXInst
-    // console.log('SUCCESS CONNECTING TO DUTCHX INSTANCE', dutchXInst)
+    dxInst = await dutchX.init(DUTCHX_OPTIONS)
```

Comment:
Actually, as we are returning `dxInst` from an async function, it's the same as returning `Promise.resolve(dxInst)`
So, no need for all that logic:
```
let promisedDX: Promise<any> 

export const initDutchXConnection = async (DUTCHX_OPTIONS: any) => {
   console.log(' ===> FIRING initDutchX ACTION')
   try {
    return promisedDX = dutchX.init(DUTCHX_OPTIONS)
  }
  ....

export { promisedDX }
```

Then there'll be no need for `getDutchXConnection`, we'll just need to `await promisedDX`
Or you can keep `export const getDutchXConnection = () => promisedDX`, if you think a function looks better semantically, no need for `async`, though.

Reply:
OK fair enough - removed async from the `getDutchXConnection` and will remove some unnecessary comments/console logs